pageserver/src/tenant/storage_layer/image_layer.rs

```diff
@@ -266,9 +268,10 @@ impl ImageLayer {
     async fn load_inner(&self, ctx: &RequestContext) -> Result<ImageLayerInner> {
         let path = self.path();
 
-        let loaded = ImageLayerInner::load(&path, self.desc.image_layer_lsn(), None, None, ctx)
-            .await
-            .and_then(|res| res)?;
+        let loaded =
+            ImageLayerInner::load(&path, self.desc.image_layer_lsn(), None, None, false, ctx)
```

Comment:
pet peeve: This call site illustrates particularly well why one should avoid naked bools and instead introduce `enum CompressedReads { Yes, No }` and pass that.